postprocess/post_b64.py

```diff
@@ -55,47 +55,18 @@ def run(results, raw_paste_data, paste_object):
                     # We are guessing that the sample has been submitted, and crafting a URL
                     paste_object["VT"] = 'https://www.virustotal.com/#/file/{0}'.format(paste_object["exe_md5"])
 
-                    # Cuckoo
-                    if conf["post_process"]["post_b64"]["cuckoo"]["enabled"]:
```

Comment:
One thing I am really concerned about is that this change will break existing configs. I'm wondering if there's any way we could make this change more backwards-compatible. I'm on the fence about if it's worth the technical debt to just say if a postprocessing module matches one of the sandboxes, print a warning and goto sandbox logic.

Reply:
I think my question would be how many users do you think would be affected by the change in config? With the change I don't think it will throw an exception on startup if you're using the old config but it would act as if no sandboxes were enabled. @kevthehermit thoughts?